.circleci/config.yml

```diff
@@ -2,37 +2,37 @@ version: 2.1
 parameters:
   GOLANG_VERSION:
     type: string
-    default: "1.17.3"
+    default: "1.17.5"
```

Comment:
CircleCI hasn't pushed the `1.17.6` [yet](https://hub.docker.com/r/circleci/golang/tags?page=1&name=1.17.6)